common/models/change.js

```diff
@@ -279,9 +279,10 @@ module.exports = function(Change) {
    */
 
   Change.hash = function(str) {
+    var buf = new Buffer(str);
     return crypto
       .createHash(Change.settings.hashAlgorithm || 'sha1')
-      .update(str)
+      .update(buf)
```

Comment:
Ok, I found the information in the commit messages. Could you please extract this change to a new pull request? Is it also possible to write a test that would fail with the current implementation in PhantomJS (the browser we use for running the tests via `npm test`)?